### PR TITLE
Feedback polish

### PR DESCRIPTION
**Pull request by CarlNelson**
* Profile polish
* Add names to check in pages to better differentiate them
* Badge polish
* Messaging for add-on expiry

### CODE REVIEW

File: lindy/check_in/templates/self_check_in/badge.html [hunk 1]
```diff
@@ -7,30 +7,30 @@
...
                     <h2>{{ user.get_full_name }}</h2>
+                    <img src="{{ user.photo.url }}" alt="Your profile photo" style="max-width: 240px;" />
                 </div>
-                <div class="columns small-8 medium-3">
-                    <img src="{{ user.photo.url }}" alt="Your profile photo" />
-                </div>
-                <div class="columns small-12 medium-7">
+                <div class="columns small-8 medium-5">
                     <h2><strong>{{ ticket.ticket_type.name }}</strong></h2>
-                    {% if group %}
```
Comment (NicoleZuckerman) [inline lindy/check_in/templates/self_check_in/badge.html]: Did you mean to get rid of the Group listing on people's badges?

**Comment by NicoleZuckerman** [comment]
Made one comment, otherwise looks good.

File: lindy/check_in/templates/self_check_in/badge.html (hunk 1 above)
Comment (CarlNelson) [inline lindy/check_in/templates/self_check_in/badge.html]: Yes, until the group stuff settles and Jaya has them relatively stable.

Comment (NicoleZuckerman) [inline lindy/check_in/templates/self_check_in/badge.html]: OK. All seems fine to me then.

**Pull request merged**